cli/src/common/utils.py

```diff
@@ -69,3 +70,12 @@ def parse_bucket_url(url):
         raise URLError("Invalid s3 url: {0}".format(url))
 
     return {"bucket_name": bucket_name, "object_key": object_key, "object_name": object_name}
+
+
+def generate_s3_bucket_hash_suffix(account_id, region):
```

Comment:
let's try to minimize the usage of the common/utils module. probably this can even be a private function for now since only used in a single place

Reply:
move into S3Bucket class as a static method.